contrib/util/language_translations/currentLanguage_utf8.sql

```diff
@@ -13357,7 +13357,8 @@ CREATE TABLE `lang_definitions` (
   `lang_id` int(11) NOT NULL default '0',
   `definition` mediumtext,
   UNIQUE KEY `def_id` (`def_id`),
-  KEY `cons_id` (`cons_id`)
+  KEY `cons_id` (`cons_id`),
+  KEY `lang_cons` (`lang_id`, `cons_id`)
```

Comment:
hi @kojiromike  , also make this change also to the file in the rel-704 branch (note the file is completely different so wouldn't cherry-pick this over. would instead to a direct modification of the file in rel-704 and push it to that branch; this will then carry over when I copy that file to the rel-800 branch in the future)